internal/configurator/testdata/curated-experience-test.expected.yaml

```diff
@@ -0,0 +1,88 @@
+scrape_configs:
+  - job_name: kubernetes-job-pod
+    kubernetes_sd_configs:
+      - role: pod
+    relabel_configs:
+      - source_labels: [__meta_kubernetes_pod_phase]
+        regex: Pending|Succeeded|Failed|Completed
+        action: drop
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scheme]
+        action: replace
+        regex: (https?)
+        target_label: __scheme__
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+        action: replace
+        target_label: __metrics_path__
+        regex: (.+)
+      - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+        action: replace
+        regex: (.+?)(?::\d+)?;(\d+)
+        replacement: $1:$2
+        target_label: __address__
+      - action: labelmap
+        regex: __meta_kubernetes_pod_annotation_prometheus_io_param_(.+)
+        replacement: __param_$1
+      - action: labelmap
+        regex: __meta_kubernetes_pod_label_(.+)
+      - source_labels: [__meta_kubernetes_namespace]
+        action: replace
+        target_label: namespace
+      - source_labels: [__meta_kubernetes_pod_node_name]
+        action: replace
+        target_label: node
+      - source_labels: [__meta_kubernetes_pod_name]
+        action: replace
+        target_label: pod
+      - source_labels: [__meta_kubernetes_pod_label_testLabel1, __meta_kubernetes_pod_label_testLabel2]
+        action: keep
+        separator: ";"
+        regex: ".*(?i)(test1|test2).*"
```

Comment:
Seems like we're filtering everything if curated experiences are on. How would a user transition from having only curated experiences enabled, to adding a particular service they are interested in?

Reply:
The idea is to have a UI similar to the following, therefore the user would need to add the services under `additional_regexes`:

```
# By default, all targets discovered are scraped.
# If curated_experience is enabled or any regex is specified,
# then merely the targets having one of the specified labels matching those regexes or the ones of the curated experiences are scraped.
# Notice that this configuration option applies to all configured kubernetes jobs.
# Under the hood relabel_configs with action 'keep' are generated, consider it in case any custom extra_relabel_config is needed
curated_experience:
  enabled: false
  source_labels: ["app.kubernetes.io/name", "k8s-app", "app"]
  additional_regexes: []
#     - ".*<regex>.*"

```

Reply:
I'm not fully sold on this UX, mainly because curated exps are enabled by default. By offering this mechanism, I think we are effectively pushing users to filter their workloads using regexes on a config file rather than labels in their cluster, which is not "the kubernetes way" of doing things.

I would prefer if we could steer users to set this "curated" filter per label. That way we could steer them to just use another label for workloads they want to monitor.

Reply:
I totally agree, the Prometheus config is decentralized, we are centralizing it. However, that is the use case 😕 

We want it to work out of the box, therefore there are not many solutions. I am not 100% getting what you are suggesting, would you like this curated experience to be applied to merely a job?

Reply:
one option (on the current solution) would be to add this labels as a wild card of the curated experience 
```
curated_experience:
  enabled: false
  source_labels: ["newrelic.io/scrape"]
  additional_regexes: [ "true" ]

```

Reply:
this would not work out of the box 😕

Reply:
Is for targets that manually want to be added by tagging. but now that i'm thinking it targets would have to have both `prometheus.io/scrape` and this additional, so is not good .

Reply:
We define a different UX during a call 😄 I'll add that and then we can discuss